era5_to_int.py

```diff
@@ -73,6 +73,83 @@ def consider(self, field, xlvl, proj, hdate, slab, intfile):
             self.t = None
             self.td = None
 
+class RHDiags:
+    """ Implements the computation of RH (relative humidity in % at pressure level)
+    from ERA5 T (pressure level temperature) and Q (specific humidity at pressure level) fields.
+    """
+    def __init__(self, levels):
+        self.spechumd = {}
+        self.t        = {}
+        self.levels   = levels
+        
+
+    def liquidSaturationVaporMixRatio(self, t, p) :
+        t0 = 273.16
+        # From MPAS src/core_atmosphere/physics/mpas_atmphys_functions.F
+        C0= .611583699E03
+        C1= .444606896E02
+        C2= .143177157E01
+        C3= .264224321E-1
+        C4= .299291081E-3
+        C5= .203154182E-5
+        C6= .702620698E-8
+        C7= .379534310E-11
+        C8=-.321582393E-13
+
+        t_bounded =np.maximum(-80.0, t - t0)
+        # Simplified calc of saturation vapor pressure
+        # esL = 612.2 * np.exp(17.67 * t_bounded / (self.t-29.65))
+        # esL with coefficients used instead
+        esL = (C0 + t_bounded *
+                  (C1 + t_bounded *
+                    (C2 + t_bounded *
+                      (C3 + t_bounded *
+                        (C4 + t_bounded *
+                          (C5 + t_bounded *
+                            (C6 + t_bounded *
+                              (C7 + t_bounded* C8)
+                            )
+                          )
+                        )
+                      )
+                    )
+                  )
+                )
+
+        # Even with P=1050mb and T=55C, the sat. vap. pres only contributes to
+        # ~15% of total pres.
+        esL = np.minimum(esL, p * 0.15)
+        mixRatioESL = .622 * esL / (p - esL)
+        return mixRatioESL
+
+    def consider(self, field, xlvl, proj, hdate, slab, intfile):
+        """ Considers whether a given field may be used in the computation
+        of RH at the surface. When all available information has been acquired,
+        this method computes the RH field and writes it to the output
+        intermediate file.
+        """
+
+        if field == 'SPECHUMD' :
+            self.spechumd[xlvl] = slab
+        # Differentiate from other TT which is at a single level
```

Comment:
This comment could be a bit confusing, since we process all fields at a single level at a time in some sense. Instead, perhaps we could explain that we don't want to consider surface pressure, whose `xlvl` value is 200100.0.